gettsim/transfers/benefit_checks/vermögens_checks.py

```diff
@@ -1,22 +1,14 @@
-import numpy as np
-import pandas as pd
-
-from gettsim.typing import BoolSeries
-from gettsim.typing import FloatSeries
-from gettsim.typing import IntSeries
-
-
 def _kinderzuschl_nach_vermög_check_m_hh(
-    _kinderzuschl_vor_vermög_check_m_hh: FloatSeries,
-    vermögen_hh: FloatSeries,
+    kinderzuschl_vorläufig_m_hh: float,
```

Comment:
I am pretty sure we explicitly changed that in the big renaming thing. Why back to `vorläufig`? It is ambiguous what is meant by that.